conda_smithy/feedstock_content/.gitignore

```diff
@@ -1,3 +1,8 @@
-*.pyc
+*
```

Comment:
Can we add a comment here explaining the overall principle of deny-by-default? I can see how some folks might be surprised by this and might go to .gitignore to check what's going.

Something like:

```
# User content belongs under recipe/.
# Feedstock configuration goes in `conda-forge.yml`
# Everything else is managed by the conda-smithy rerender process. 
# Please do not modify.
```